plex-service/src/main/java/com/cleverpine/plex/entity/future/EpisodeEntity.java

```diff
@@ -0,0 +1,74 @@
+package com.cleverpine.plex.entity.future;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+
+import java.time.LocalDate;
+
+@Getter
+@Entity
+@Table(name = "episodes")
```

Comment:
Look up the Lombok annotations for the class constructors and use them instead of writing them. It saves space and it's more readable.